public/src/editor/media-gallery-editor.js

```diff
@@ -343,11 +343,16 @@ define( [ "localized", "util/lang", "util/uri", "util/xhr", "util/keys", "util/m
       _errorMessage.classList.remove( "hidden" );
       _addMediaPanel.classList.add( "invalid-field" );
     }, MEDIA_LOAD_TIMEOUT );
-    MediaUtils.getMetaData( data.source, onSuccess, function() {
-      clearTimeout( _cancelSpinner );
-      clearTimeout( _mediaLoadTimeout );
-      deactivateSpinner();
-      pagingSearchCallback( _itemContainers.project, 1 );
+    MediaUtils.getMetaData( data.source, onSuccess, function( errorMessage ) {
+      // Unplayable clip, so display search term pages.
+      if (errorMessage === "This media source is unplayable") {
```

Comment:
Lame, we should translate *right* before spitting it into UI.

Localized error messages in callbacks is kinda silly. Ahh well.